taxi/views.py

```diff
@@ -87,3 +88,27 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreate(LoginRequiredMixin, generic.CreateView):
+    form_class = DriverCreateForm
+    model = Driver
+
+
+class DriverLicenseUpdateForm(LoginRequiredMixin, generic.UpdateView):
+    form_class = DriverLicenseUpdateForm
+    model = Driver
+
+
+class DriverDelete(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+@login_required
+def add_and_remove_driver(request, car_id):
+    car = Car.objects.get(pk=car_id)
+    if request.method == "POST":
+        car.drivers.add(request.user)
+        return redirect("taxi:car-detail", pk=car_id)
+    return redirect("taxi:car-detail", pk=car_id)
```

Comment:
let's make it as class-based view as well. Also don't see where is remove action here